<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>1</y>
      <width>651</width>
      <height>420</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListWidget" name="listWidget">
           <item>
            <property name="text">
             <string>所有文献</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>最近添加</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>收藏夹</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>未分类</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>作者过滤</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>自定义Tag过滤</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>出版物过滤</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>标签</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>详情</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>笔记</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="menu_action_add_file"/>
    <addaction name="menu_action_manual_input"/>
    <addaction name="separator"/>
    <addaction name="menu_action_remove_file"/>
    <addaction name="menu_action_rename_file"/>
    <addaction name="separator"/>
    <addaction name="menu_action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="menu_action_undo"/>
    <addaction name="menu_action_redo"/>
    <addaction name="separator"/>
    <addaction name="menu_action_search"/>
    <addaction name="separator"/>
    <addaction name="menu_action_copy"/>
    <addaction name="menu_action_cut"/>
    <addaction name="menu_action_paste"/>
    <addaction name="separator"/>
    <addaction name="menu_action_select_all"/>
    <addaction name="separator"/>
    <addaction name="menu_action_add_category"/>
    <addaction name="menu_action_rename_category"/>
    <addaction name="menu_action_remove_category"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图</string>
    </property>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>列表视图</string>
     </property>
     <addaction name="menu_action_show_in_table"/>
     <addaction name="menu_action_show_in_icon"/>
    </widget>
    <addaction name="menu_action_show_details"/>
    <addaction name="menu_6"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="menu_action_options"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="menu_action_user_manual"/>
    <addaction name="separator"/>
    <addaction name="menu_action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menu_action_add_file">
   <property name="text">
    <string>添加文件</string>
   </property>
  </action>
  <action name="menu_action_remove_file">
   <property name="text">
    <string>删除文档</string>
   </property>
  </action>
  <action name="menu_action_rename_file">
   <property name="text">
    <string>重命名文献</string>
   </property>
   <property name="toolTip">
    <string>重命名文献</string>
   </property>
  </action>
  <action name="menu_action_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="menu_action_undo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="menu_action_redo">
   <property name="text">
    <string>反撤销</string>
   </property>
  </action>
  <action name="menu_action_search">
   <property name="text">
    <string>搜索</string>
   </property>
  </action>
  <action name="menu_action_copy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="menu_action_paste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="menu_action_cut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="menu_action_select_all">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="menu_action_add_category">
   <property name="text">
    <string>新建分类</string>
   </property>
   <property name="toolTip">
    <string>新建分类</string>
   </property>
  </action>
  <action name="menu_action_rename_category">
   <property name="text">
    <string>重命名分类</string>
   </property>
   <property name="toolTip">
    <string>重命名分类</string>
   </property>
  </action>
  <action name="menu_action_remove_category">
   <property name="text">
    <string>删除分类</string>
   </property>
   <property name="toolTip">
    <string>删除分类</string>
   </property>
  </action>
  <action name="menu_action_show_details">
   <property name="text">
    <string>显示文献详情</string>
   </property>
  </action>
  <action name="menu_action_show_in_table">
   <property name="text">
    <string>表格显示</string>
   </property>
  </action>
  <action name="menu_action_show_in_icon">
   <property name="text">
    <string>图标显示</string>
   </property>
  </action>
  <action name="menu_action_options">
   <property name="text">
    <string>选项</string>
   </property>
  </action>
  <action name="menu_action_user_manual">
   <property name="text">
    <string>用户手册</string>
   </property>
  </action>
  <action name="menu_action_about">
   <property name="text">
    <string>关于文献管理</string>
   </property>
  </action>
  <action name="menu_action_manual_input">
   <property name="text">
    <string>手工录入</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>menu_action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
